scaleway/resource_instance_security_group_rules.go

```diff
@@ -35,40 +36,27 @@ func resourceScalewayInstanceSecurityGroupRules() *schema.Resource {
 				Description: "Outbound rules for this set of security group rules",
 				Elem:        securityGroupRuleSchema(),
 			},
-			"zone":            zoneSchema(),
```

Comment:
ok to remove org ID, but zone might be required

Use case:
```hcl
resource scaleway_instance_security_group_rules sgrs01 {
	security_group_id = "123456-789-abc-def" // handwritten UUID
	zone = nl-ams-1 // security group is in a different zone than default zone
} 
```

Reply:
`security_group_id` is a zonedID, computed from `scaleway_instance_security_group.id()`

Reply:
There was a misunderstanding about the way to handle child resources. Child resource MUST rely on a parent ID that includes the locality `fr-par-1/xxxx-xx-xx-xx`, which means:
- let's discard this comment
- we must document this clearly (not the scope of this PR)
- we must review all child resources to make sure we do this consistently (not the scope of this PR)